specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -2015,6 +2019,10 @@
       "description": "Detailed information about the status if any",
       "type": "object",
       "properties": {
+        "recipientMailServer": {
```

Comment:
Should this be something like recipientMailServerHostName?

Reply:
recipientMailServer is used internally and our public facing documentation in azure monitor logs. So this should be fine.

Reply:
The fact that this name is used internally doesn't really change how we think about naming for public APIs.